### DISCUSSION TITLE
CouchDB Integration?

### DISCUSSION THREAD

**Discussion by fynnlyte** [Budibase/budibase#4118] [General]
Hi,

I see that budibase uses CouchDB internally and that you're planning to also [integrate it as database source](https://budibase.com/integration/couchdb/). 

Is there any ETA on that? Or something that I could do myself if I want to try out budibase with per-user databases on CouchDB as data source?

**Comment by mjashanks** [comment]
Hey @LyteFM - we have a CouchDB integration already. You can add it as a datasource!

**Reply by fynnlyte** [reply]
Oh that's great news! Maybe you'll want to adjust the [documentation](https://budibase.com/integration/couchdb/) accordingly, then. I thought „cooming soon” meant that the integration is not available yet ;)

**Reply by mjashanks** [reply]
Of course :/ Thanks! Work to do on the docs :)
